src/GridFieldAddNewInlineButton.php

```diff
@@ -86,11 +86,11 @@ public function getHTMLFragments($grid)
 
         $fragment = $this->getFragment();
 
-        if (!$editable = $grid->getConfig()->getComponentByType('GridFieldEditableColumns')) {
+        if (!$editable = $grid->getConfig()->getComponentByType(GridFieldEditableColumns::class)) {
             throw new Exception('Inline adding requires the editable columns component');
         }
 
-        Requirements::javascript(THIRDPARTY_DIR . '/javascript-templates/tmpl.js');
+        Requirements::javascript('gridfieldextensions/javascript/tmpl.js');
```

Comment:
@dhensby got some documentation on that new resource loader? Didn't see in the docs. Is it just `Requirements::javascript('symbiote/silverstripe-gridfieldextensions:javascript/tmpl.js')` or do you have to use the full example as per the changelog

```
use SilverStripe\View\ThemeResourceLoader;
$moduleFilePath = ModuleLoader::getModule('silverstripe/framework')->getRelativeResourcePath('MyFile.php');
```